Sources/StytchCore/Documentation.docc/StytchCore.md

```diff
@@ -10,111 +10,64 @@ _To see in-depth examples of basic, intermediate, and advanced usage of the Styt
+``StytchB2BClient/OAuth-swift.struct`` | ``StytchB2BClient/OAuth-swift.struct/ThirdParty/start(configuration:)``, ``StytchB2BClient/OAuth-swift.struct/authenticate(parameters:)-80abl``
+``StytchB2BClient/OTP-swift.struct`` | ``StytchB2BClient/OTP/send(parameters:)-3xcc9``, ``StytchB2BClient/OTP/authenticate(parameters:)-5ums0``
+``StytchB2BClient/TOTP-swift.struct`` | ``StytchB2BClient/TOTP/create(parameters:)-437r4``, ``StytchB2BClient/TOTP/authenticate(parameters:)-2ck6w``
+``StytchB2BClient/Passwords-swift.struct`` | ``StytchB2BClient/Passwords-swift.struct/authenticate(parameters:)-63kup``, ``StytchB2BClient/Passwords-swift.struct/resetByEmailStart(parameters:)-24ggc``, ``StytchB2BClient/Passwords-swift.struct/resetByEmail(parameters:)-6r4gk``, ``StytchB2BClient/Passwords-swift.struct/resetByExistingPassword(parameters:)-2ju8w``, ``StytchB2BClient/Passwords-swift.struct/resetBySession(parameters:)-834cf``, ``StytchB2BClient/Passwords-swift.struct/strengthCheck(parameters:)-4uctk``
+``StytchB2BClient/RBAC-swift.struct`` | ``StytchB2BClient/RBAC-swift.struct/allPermissions()-89p7d``, ``StytchB2BClient/RBAC-swift.struct/isAuthorized(resourceId:action:)-3qmjb``, ``StytchB2BClient/RBAC-swift.struct/isAuthorizedSync(resourceId:action:)``
+``StytchB2BClient/RecoveryCodes-swift.struct`` | ``StytchB2BClient/RecoveryCodes-swift.struct/get()-1dlsm``, ``StytchB2BClient/RecoveryCodes-swift.struct/rotate()-9wyz3``, ``StytchB2BClient/RecoveryCodes-swift.struct/recover(parameters:)-7r6fr``
+``StytchB2BClient/SearchManager-swift.struct`` | ``StytchB2BClient/SearchManager-swift.struct/searchMember(searchMemberParameters:)-9nw94``, ``StytchB2BClient/SearchManager-swift.struct/searchOrganization(searchOrganizationParameters:)-2a7yp``
+
 
 ## Topics
 
 ### Consumer
 
 - ``StytchClient``
+- ``AuthenticateResponseData``
 - ``Session``
 - ``User``
 
 ### B2B
 
 - ``StytchB2BClient``
-- ``Organization``
+- ``B2BAuthenticateResponseData``
+- ``B2BMFAAuthenticateResponseData``
 - ``MemberSession``
+- ``Organization``
 - ``Member``
 
-### Sessions
+### Common
 
-- ``Sessions``
 - ``SessionToken``
-
-### Authenticate Response Types
-- ``AuthenticateResponse``
-- ``AuthenticateResponseType``
-- ``AuthenticateResponseDataType``
-- ``AuthenticateResponseData``
-- ``B2BAuthenticateResponse``
-- ``B2BAuthenticateResponseType``
-- ``B2BAuthenticateResponseDataType``
-- ``B2BAuthenticateResponseData``
-
-### Generic Response Types
-
-- ``BasicResponseType``
-- ``BasicResponse``
-- ``Response``
-
-### Error Types
-- ``StytchError``
-
-### Deeplink Types
-
-- ``DeeplinkHandledStatus``
-
-### Additional Types
-
-- ``AuthenticationFactor``
-- ``SSORegistration``
-- ``Identifier``
-- ``Completion``
-- ``Minutes``
-- ``EmptyCodable``
-- ``JSON``
-- ``Union``
-- ``UserResponse``
+- ``SessionTokens``
+- ``RBACPolicy`
+- ``Response`
+- ``StytchError`
+- ``StytchAPIError`
+- ``PKCECodePair`
```

Comment:
Are the above five items missing a "`" at the end?

Reply:
great catch!